qubesadmin/backup/restore.py

```diff
@@ -1459,9 +1464,10 @@ def restore_info_verify(self, restore_info):
 
             if not self.options.verify_only and \
                     vm_info.name in self.app.domains:
```

Comment:
This adds prefix only to VMs that already exists on the system. I guess it isn't what you wanted.